manifests/00-custom-resource-definition.yaml

```diff
@@ -1,24 +1,33 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  creationTimestamp: null
   name: ingresscontrollers.operator.openshift.io
 spec:
   group: operator.openshift.io
   names:
     kind: IngressController
-    listKind: IngressControllerList
```

Comment:
@ironcladlou is there a reason why the `listKind` field was removed?

Reply:
The `listKind` is now inferred by convention relative to `kind`.